contracts/HydraChain/modules/DaoIncentive/DaoIncentive.sol

```diff
@@ -17,9 +17,9 @@ abstract contract DaoIncentive is
     RewardWalletConnector,
     HydraStakingConnector
 {
-    address public daoIncentiveVaultAddr;
+    uint256 public lastDistribution;
```

Comment:
Why didn't you leave the comment and even improve it by making it in natspec format?

Reply:
I think the variable name itself tells enough. Also if a dev check the function the logic, it will be very clear.